src/tests/functional/plugin/shared/src/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -91,7 +91,6 @@ void OVPropertiesTestsWithCompileModelProps::SetUp() {
     }
 
     if (target_device == std::string(ov::test::utils::DEVICE_MULTI) ||
```

Comment:
this whole if can be removed now, adding MULTI_DEVICE_PRIORITIES is now handled by the --config

Reply:
@sbalandi Will you update the conformance test for sw plugin to use --config_path= in the conformance test in pre-commit/commit and nightly? Or use AUTO:TEMPLATE for test?

Reply:
--config_path have already been used in nightly validation; conformance with sw plugins is not run in pre_commit, only cpu conformance